runtime/jcl/common/stdinit.c

```diff
@@ -111,6 +111,9 @@ standardInit( J9JavaVM *vm, char *dllName)
 
 	TOC_STORE_TOC(vm->jclTOC, standardInit);
 
+	memset(&jclFakeClass, 0, sizeof(jclFakeClass));
+	/* the fake class must be exempt because it references package-private classes */
+	jclFakeClass.classFlags = J9ClassIsExemptFromValidation;
```

Comment:
This should also be unnecessary since we're not removing unsafe.

Reply:
Reverted.